features/messages/impl/src/main/kotlin/io/element/android/features/messages/impl/voicemessages/timeline/VoiceMessagePresenter.kt

```diff
@@ -52,12 +58,23 @@ interface VoiceMessagePresenterModule {
     fun bindVoiceMessagePresenterFactory(factory: VoiceMessagePresenter.Factory): TimelineItemPresenterFactory<*, *>
 }
 
+class VoiceMessagePresenterState {
+    val play = mutableStateOf<Async<Unit>>(Async.Uninitialized)
+    var progress by mutableFloatStateOf(0f)
+}
+
 class VoiceMessagePresenter @AssistedInject constructor(
     voiceMessagePlayerFactory: VoiceMessagePlayer.Factory,
     private val analyticsService: AnalyticsService,
+    private val timelineItemPresenterFactories: TimelineItemPresenterFactories,
+    private val scope: CoroutineScope,
```

Comment:
If you are using the app coroutine scope, then if you close the room screen it'll still be playing?